cypher/cypher-docs/src/docs/dev/ql/administration/security/user-management-syntax.asciidoc

```diff
@@ -64,38 +56,36 @@ CREATE OR REPLACE USER name
 | Create a new user, or if a user with the same name exists, replace it.
 | <<administration-security-administration-dbms-privileges-user-management, CREATE USER>> and
 <<administration-security-administration-dbms-privileges-user-management, DROP USER>>
-| `+`
-| `+`
 
-| [source, cypher]
+| [source]
 ----
 ALTER USER name SET {
-[PLAINTEXT \| ENCRYPTED] PASSWORD password
-            [[SET PASSWORD] CHANGE [NOT] REQUIRED]
-            [SET STATUS {ACTIVE \| SUSPENDED} ] \|
+  [PLAINTEXT \| ENCRYPTED] PASSWORD password
+  [[SET PASSWORD] CHANGE [NOT] REQUIRED]
+  [SET STATUS {ACTIVE \| SUSPENDED} ]
+  [SET DEFAULT DATABASE databaseName] \|
 PASSWORD CHANGE [NOT] REQUIRED
-            [SET STATUS {ACTIVE \| SUSPENDED}] \|
+  [SET STATUS {ACTIVE \| SUSPENDED}]
+  [SET DEFAULT DATABASE databaseName] \|
 STATUS {ACTIVE \| SUSPENDED}
-            [SET DEFAULT DATABASE databaseName]
+  [SET DEFAULT DATABASE databaseName] \|
+DEFAULT DATABASE databaseName
+}
```

Comment:
Ah you found the missing `}` as well :P I have added it in my clean-up PR as well (and was what I conflicted on when I found the errors)